pkg/controller/controller.go

```diff
@@ -617,8 +617,8 @@ func (p *csiProvisioner) ProvisionExt(options controller.ProvisionOptions) (*v1.
 		}
 	}
 
-	klog.Infof("successfully created PV %+v", pv.Spec.PersistentVolumeSource)
-
+	klog.Infof("successfully created PV %v for PVC %v driver name %v csi volume name %v", pv.Name, options.PVC.Name, pv.Spec.CSI.Driver, pv.Spec.CSI.VolumeHandle)
```

Comment:
Explicitly set a log level here.  How about 2?

Reply:
I don't think we need to log driver name here? The external-provisioner only handles one driver.

Reply:
yes you  are right, removed driver name logging and set the log level to 2